### ISSUE TITLE
Shows translations of "game", "mod", "texture pack" in lowercase

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/contentdb#355]
## Summary
In some cases, the words "game", "mod" and "texture pack" in the translated versions are always written in lowercase, even if the translation starts with an uppercase letter.

This is the case in the searchbar ("Search X") or the rank ("Y has a game ranked #1") info and possibly other places.

Example: In German, what should be “Spiel” (game) becomes “spiel”.

**Comment by rubenwardy** [comment]
What are the rules for whether it should be uppercase or lowercase? In English, it's just the first letter of a sentence and proper nouns.

Types are currently translated with two strings, one for singular and one for plural.

Examples of singular:

```
A release is a single downloadable version of your game.
Top game
Top 5 game
You can include a .cdb.json file in your game to update these details automatically.
```

Example of plural:

```
Search games
```

The code currently does `.lower()` on the types inserted into a sentence in the middle

**Comment by rubenwardy** [comment]
The places where ContentDB doesn't lower the type name includes:

* The featured jumbotron on the homepage
* `Type: Game` on the package page sidebar
* In the similar packages page, as `[Game]`

**Comment by Wuzzy2** [comment]
In general, just don't make any blind assumptions about other languages if you don't know them. You will just stumble again and again and again, lol.

Assuming that certain words are always in lowercase in an unknown latin-script language is wrong. Automatically forcing lower case is VERY wrong.

One solution would be to provide additional strings, depending on whether it's a "title word" or a word in a sentence.

As for German, all nouns are uppercase, so it's always "Spiel" (for "game"), not "spiel".

**Comment by rubenwardy** [comment]
It's very hard not to make assumptions, especially when translation was added to ContentDB after the fact

I'll add variations based on whether it's a title word or in a sentence

**Comment by Wuzzy2** [comment]
A better (and maybe simpler?) solution: Add a separate full sentence string for each content type instead of inserting single words into the sentence. As there are only 3 (or so) content types, this should be OK. I do this strategy for my projects all the time. This also solves possible grammar problems in certain languages.

So like, don't make the string "Do you recommend this %s?" (where %s is "game", "mod" or "texture pack") translatable, but make 3 sentences "Do you recommend this game?", "Do you recommend this mod?" and "Do you recommend this texture pack?".

**Comment by Wuzzy2** [comment]
Thanks for your work! As soon I saw this update, I immediately went to work and updated German. I also saw the update a few minutes later, so I believe the changes are now live.

But this issue is still not fully fixed.

This string still writes the content type in lowercase:

> Do you recommend this %(type)s?

In German becomes:

> Empfehlen Sie dieses spiel?

Although the word "game" is translated "Spiel" in German.
